docs/source/workflow_outputs.rst

```diff
@@ -462,3 +463,31 @@ Design loads are based on block load ACCA Manual J calculations.
    Design Loads Cooling Latent: Infiltration/Ventilation (Btu/h)     Latent cooling design load for infiltration/ventilation
    Design Loads Cooling Latent: Internal Gains (Btu/h)               Latent cooling design load for internal gains
    ================================================================  ====================
+
+.. _bill_outputs:
+
+Utility Bill Outputs
+--------------------
+
+OpenStudio-HPXML can optionally generate a utility bills output file.
+The utility bills output file is called ``results_bills.csv`` (or ``results_bills.json``) and located in the run directory.
+Monthly fixed charges and marginal rates, as well as PV compensation types/rates/fees, are defined to determine how utility bills are calculated.
+
+The file includes:
+
+   =============================================  ====================
+   Type                                           Notes
+   =============================================  ====================
+   Electricity: Fixed ($)                         Annual fixed charges for electricity.
```

Comment:
Can we also add a simple "Total ($)", which is the sum of all the fuel totals, to the results_bills.csv and document that here?